daemon/config/config.go

```diff
@@ -351,8 +352,21 @@ func getConflictFreeConfiguration(configFile string, flags *pflag.FlagSet) (*Con
 	}
 
 	reader = bytes.NewReader(b)
-	err = json.NewDecoder(reader).Decode(&config)
-	return &config, err
+	if err := json.NewDecoder(reader).Decode(&config); err != nil {
```

Comment:
Yes, I added that commit based on this finding :smile: Was writing it up as an example, and decided to save @jlhawn the time and add rebase it, and add it to the PR